zkevm-circuits/src/witness/block.rs

```diff
@@ -50,6 +51,24 @@ pub struct Block<F> {
     pub keccak_inputs: Vec<Vec<u8>>,
     /// Original Block from geth
     pub eth_block: eth_types::Block<eth_types::Transaction>,
+
```

Comment:
I'm wondering about the naming of the structs.  This struct is called `Block`, but it has fields that are for chunks.
Would it make sense to take a `Block` and from it generate `n` `BlockChunk` types? and all the fingerprint fields would only appear in the `BlockChunk` struct.

Then a update all the circuits to take a `BlockChunk` instead of a `Block`?

Another option is for the `Block` to contain a field called `chunks` with a vector of `Chunks` which has the chunk specific fields (like the fingerprints).  Then each circuit takes a `Block` and a `chunk_index`.

Reply:
Thanks for the suggestion!
Right now the naming and place is naively design  
I plan to refactor this part in later multiple chunks PRs, at that moment we will be more clear on how to design this part. Wdyt :) ?

Reply:
Sounds good!